db/storage/admin-files.ts

```
@@ -0,0 +1,133 @@
+  let validFilename = fileRecord.name
+    .replace(/[^a-z0-9.]/gi, '_')
+    .toLowerCase();
+  const extension = validFilename.split('.').pop();
+  const baseName = validFilename.substring(0, validFilename.lastIndexOf('.'));
+  const maxBaseNameLength = 100 - (extension?.length || 0) - 1;
+  if (baseName.length > maxBaseNameLength) {
+    validFilename = `${baseName.substring(0, maxBaseNameLength)}.${extension}`;
+  }
+  fileRecord.name = validFilename;
+
+  let createdFile;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)